packages/components/src/components/install-guide/record-instructions/Java.vue

```diff
@@ -0,0 +1,179 @@
+<template>
+  <section>
+    <p>
+      Use the "Run with AppMap" debug configuration to launch your application or run your tests
```

Comment:
Is it a debug configuration, or a launch configuration? I thought it was the latter.